src/server_manager/web_app/ui_components/if_messages.ts

```diff
@@ -0,0 +1,35 @@
+/*
+  Copyright 2018 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import {nothing, LitElement, html} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+
+@customElement('if-messages')
+export class IfMessages extends LitElement {
+  @property({type: String, attribute: 'message-ids'}) messageIDs = '';
+  @property({type: Function, attribute: 'localize'}) localize: (
+    msgId: string,
+    ...params: string[]
+  ) => string;
+
+  render() {
+    if (this.messageIDs.split(/,\s*/).some((id) => this.localize(id) === id)) {
```

Comment:
Let's make the message check more robust by also checking against undefined or empty string.